lib/system.ml

```diff
@@ -56,6 +56,13 @@ let exists_dir dir =
   let dir = if Sys.os_type = "Win32" then strip_trailing_slash dir else dir in
   try Sys.is_directory dir with Sys_error _ -> false
 
+let rec mkdirhier dir =
```

Comment:
I think this should just be called `mkdir`.

Reply:
Add that it creates the entire subdirectory in the mli.

Reply:
I used mkdirhier in reference to the Unix command of same name (but I agree that mkdir is simpler).